bin/edit.js

```diff
@@ -26,6 +35,10 @@ function handleEdit(args) {
     var wtName  = args.name ? args.name + '/' : '';
     var url     = profile.url + '/edit/' + profile.container + '#/' + wtName + profile.token;
 
+    if (args.previousVersion) {
+        url = profile.url + '/prev/edit/' + profile.container + '#/' + wtName + profile.token;
```

Comment:
Is there a problem with optionally allowing the specification of minor and patch as well? I'd think this would be a valuable workaround if we introduce unintended regression for example. But if this is problematic, I think choosing a major only is good enough.